src/PeriodEnd/ManualAdjustments/SFA.DAS.ProviderPayments.Calc.ManualAdjustments.IntegrationTests/Specs/WhenProcessingAnAdjustmentForAnEmployerAccount.cs

```diff
@@ -143,8 +143,8 @@ public void ThenItShouldAdjustEmployersAccountBalance()
         public void ThenItShouldCreateReversingRequiredPaymentForTwoPayments()
```

Comment:
remove unused using

Reply:
remove redundant method args

Reply:
AssertResults has a boolean assertCollectionPeriod which is set to true and no other usages of it. Should remove the argument and its usages.